### ISSUE TITLE
\e[2m not working properly ?

### ISSUE THREAD

**Issue by lonnywong** [asciinema/agg#66]
**Describe the bug**
Check https://github.com/asciinema/asciinema-player/issues/52

**To Reproduce**
1. Run the script and covert it to gif:
```sh
#!/bin/bash

printf "\n"
printf "    _   __                           __   \e[2m______      _       __ \e[0m\n"
printf "   / | / /___  _________ ___  ____ _/ /  \e[2m/ ____/___ _(_)___  / /_\e[0m\n"
printf "  /  |/ / __ \/ ___/ __ ´__ \/ __ ´/ /  \e[2m/ /_  / __ ´/ / __ \/ __/\e[0m\n"
printf " / /|  / /_/ / /  / / / / / / /_/ / /  \e[2m/ __/ / /_/ / / / / / /_  \e[0m\n"
printf "/_/ |_/\____/_/  /_/ /_/ /_/\__,_/_/  \e[2m/_/    \__,_/_/_/ /_/\__/  \e[0m\n"
printf "\n"
```

**Expected behavior**
The gif works same as asciinema player.

**Versions:**
 - OS: macOS 14.2.1
 - agg: 1.4.3 89c957608f44d3450335120f89222ac138929f91

**Comment by ku1ik** [comment]
Thanks. This is missing indeed, but should be rather straightforward to add faint support.

**Comment by ku1ik** [comment]
Faint attr support is now available in the default (fontdue) renderer. This will be released with the next version, very soon.

We'll be able to also implement that in the alternative renderer (resvg) once opacity is supported on tspan elements in resvg (https://github.com/RazrFalcon/resvg/issues/414).

**Comment by ku1ik** [comment]
So, I released 1.5.0 with this improvement (amongst many others). I made resvg renderer a default, as it gives overall better results (better font rendering, emoji, etc). Faint attr is not supported in it, but you can run `agg --renderer fontdue ...` to use fontdue renderer, which supports faint rendering.
